tts-python-advanced-customization-with-ssml.ipynb

```diff
@@ -317,6 +343,104 @@
     "    print(\"--------------------------------------------\")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Replacing pronunciation with the `sub` tag\n",
+    "\n",
+    "We can use the `sub` tag to replace the pronounciation of the specified word or phrase with a different word or phrase. You can specify the pronunciation to substitute with the alias attribute.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "# Setting up Riva TTS request with SynthesizeSpeechRequest\n",
+    "\"\"\"\n",
+    "    Raw text is \"WWW is know as the web\"\n",
+    "    We are updating this raw text with SSML:\n",
+    "    1. Envelope raw text in '<speak>' tags as is required for SSML\n",
+    "    2. Add '<sub>' tag with 'alias' attribute set to replace www with `World Wide Web`\n",
+    "\n",
+    "\"\"\"\n",
+    "raw_text = \"WWW is know as the web.\"\n",
+    "ssml_text = '<speak><sub alias=\"World Wide Web\">WWW</sub> is known as the web.</speak>'\n",
+    "\n",
+    "print(\"Raw Text: \", raw_text)\n",
+    "print(\"SSML Text: \", ssml_text)\n",
+    "\n",
+    "sample_rate_hz = 44100\n",
+    "# Request to Riva TTS to synthesize audio\n",
+    "resp = riva_tts.synthesize(\n",
+    "    text=ssml_text,\n",
+    "    voice_name=\"English-US.Female-1\",\n",
+    "    language_code=\"en-US\",\n",
+    "    encoding=riva.client.AudioEncoding.LINEAR_PCM,\n",
+    "    sample_rate_hz=sample_rate_hz,\n",
+    ")\n",
```

Comment:
Let's clean this notebook up. We don't have to re-define this everytime. Let's just define this once at the beginning of the notebook and change the text field. `resp.text = ssml_text`

Can we do the pip install of riva-client in the notebook itself?

Reply:
I can set up a dictionary and pass it as arguments.
>Can we do the pip install of riva-client in the notebook itself?
I need to try this out